### PR TITLE
Removed unused functions.

### PR DESCRIPTION
**Pull request by baloyan**
- Removed SerilizeTicketsBundle() and DeserilizeTicketsBundle() functions, because not TicketsBundle object has it's own SerializeToJSON() and DeserilizeFromJSON() methods.
- Removed SerilizeBatchSignature() and DeserilizeBatchSignature() functions, because not BatchSignature object has it's own SerializeToJSON() and DeserilizeFromJSON() methods.

### CODE REVIEW

**Comment by vladimir-v-diaz** [comment]
I can help migrate the review comments for this pull request from baloyan/CacheCash to here.

File: include/common/utils.h [hunk 1]
```diff
@@ -37,53 +38,6 @@ void ExtractPrivateKey(const char* filename, uint8_t privkey[]);
...
-                            uint32_t &length);
-
-
-// Deserialize the batch signature from a bytes stream received over the network.
-//
-// Arguments:
-// - BatchSignature type object
-// - Plain message, or coming message part two
-// - Message length, which will be calculated inside of this function
-void DeserializeBatchSignature(BatchSignature &batch_sig,
```
Comment (vladimir-v-diaz) [inline include/common/utils.h]: `src/data_types/tickets_bundle.cpp` and `src/data_types/tickets_bundle.cpp` appear to  both call DeserializeBatchSignature().  Do you remove these calls in a different pull request?

Artiom: Yes, I removed function calls in previous PRs #1 and #2. In current PR I just removed unused functions.

File: src/common/utils.cpp [hunk 2]
```diff
@@ -93,231 +94,6 @@ void ExtractPublicKey(const char* filename, uint8_t pubkey[])  {
...
-// network.
-// NOTE: the reading does not include the type and size fields, we need to read
-// them first before being able to read the bundle itself. This reading does
-// not include the batch signature part.
-//
-// Arguments:
-// - TicketsBundle type object
-// - Plain message or coming message part two
-// - Message length, which will be calculated inside of this function
-void DeserializeTicketsBundle(TicketsBundle* tkts_bundle,
```
Comment (vladimir-v-diaz) [inline src/common/utils.cpp]: Similarly, DeserializeTicketsBundle() appears to be called elsewhere (e.g., src/data_types/tickets_bundle.cpp). Please verify that functions removed in this pull request are no longer being called by the code (or state the pull request that removes these calls).

**Comment by vladimir-v-diaz** [comment]
FYI: There are a bunch of typos in your pull request summary (e.g., SerilizeTicketsBundle() and TicketsBumdle), which you don't really have to fix. Fortunately, these typos were not present in the code, at least in the code/functions being removed in this pull request and a hasty use of grep.

**Comment by vladimir-v-diaz** [comment]
As we discussed in person, I will instead review the preceding pull requests first, particularly the http_connection_17" pull request.

File: include/common/utils.h (hunk 1 above)
Comment (baloyan) [inline include/common/utils.h]: Yes, I removed function calls in previous PRs #1 and #2. In current PR I just removed unused functions.

**Comment by baloyan** [comment]
This PR was reviewed by @vladimir-v-diaz . Before merging please make sure the previous PRs have already been reviewed and merged with master.
